assets/ocp-performance/panels.libsonnet

```diff
@@ -26,8 +26,21 @@ local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonn
       + options.legend.withShowLegend(true)
       + options.legend.withCalcs([
         'mean',
+        'min',
         'max',
+      ])
+      + options.legend.withSortBy('Max')
+      + options.legend.withSortDesc(true)
+      + options.legend.withPlacement('bottom'),
+
+    genericLegendCounter(title, unit, targets, gridPos):
```

Comment:
New panel type with different legend fields, more relevant for counters and memory.